src/m365/spo/commands.ts

```diff
@@ -247,6 +247,7 @@ export default {
   SITE_APPPERMISSION_LIST: `${prefix} site apppermission list`,
   SITE_APPPERMISSION_REMOVE: `${prefix} site apppermission remove`,
   SITE_APPPERMISSION_SET: `${prefix} site apppermission set`,
+  SITE_ARCHIVE: `${prefix} site archive`,
```

Comment:
Thinking of it some more, I think this command should be called `spo tenant site archive`. Let's discuss this on the issue.

Reply:
I agree. I will wait for the discussion to conclude before making this change and marking the PR as ready.